docs/team.mdx

```diff
@@ -6,29 +6,23 @@ title: Zenith Team
 The Zenith Emerging Technologies Special Interest Group (Zenith SIG) co-chairs are:
 
 <Bio name="Keith O'Donnell" organisation="Feynic" image="/img/people/keith-o-donnell.png">
-Keith O’Donnell is the Managing Director of Feynic Technology Ltd, an advisory consultancy focused on the commercialisation of emerging technologies. 
-
-He previously led the EMEA Innovation Program for Morgan Stanley, seeding their Spatial Computing & Quantum Technology programs and developing numerous proof-of-concept projects across a number of disciplines relevant to this special interest group proposal.
+Keith O’Donnell is the Managing Director of Feynic Technology Ltd, an advisory consultancy focused on the commercialisation of emerging technologies. He previously led the EMEA Innovation Program for Morgan Stanley, seeding their Spatial Computing & Quantum Technology programs and developing numerous proof-of-concept projects across a number of disciplines relevant to this special interest group proposal.
 
 Working across numerous industries since 2001, Keith has earned a varied skillset that enables him to adapt quickly to new technologies and disciplines. This experience covers the construction, logistics, finance and technology sectors. He currently holds an affiliate position with the Scottish International Space Advisory Council (SISAC) and is an active member of the Global Scot network.
 
 Keith earned his degree in Business from the University of Strathclyde, having previously studied Genetics & Biotechnology at the University of Glasgow.
 </Bio>
 
 <Bio name="Patrick Downing" organisation="Morgan Stanley" image="/img/people/patrick-downing.png">
-Patrick Downing is a Vice President of Identity and Access Management focusing on strategic, core infrastructure initiatives across Morgan Stanley. 
-
-He has helped establish and now leads the Quantum Technology efforts at Morgan Stanley, focusing on near-term use case identification and implementation. 
+Patrick Downing is a Vice President of Identity and Access Management focusing on strategic, core infrastructure initiatives across Morgan Stanley. He has helped establish and now leads the Quantum Technology efforts at Morgan Stanley, focusing on near-term use case identification and implementation. 

```

Comment:
It just kicked the spacing issue down a paragraph. Think there is something more structural at foot.

![image](https://github.com/finos/zenith/assets/19316277/0a075f4c-17ad-4b10-95fe-901116a6574b)

Reply:
Strange, it's a normal {children} tag